parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -43,6 +43,10 @@ createType
     : CREATE (OR REPLACE)? (EDITIONABLE | NONEDITIONABLE)? TYPE plsqlTypeSource
     ;
 
+createTrigger
+    : CREATE (OR REPLACE)? TRIGGER triggerName (BEFORE | AFTER | INSTEAD OF) dmlEventClause
```

Comment:
Hi can we put the definition of this syntax in the PLSQL.g4 file?
![2023-12-31_16-07](https://github.com/apache/shardingsphere/assets/106047330/cf259483-e8f6-47f9-b82c-2e2122c2450a)

Syntax ref: https://docs.oracle.com/en/database/oracle/oracle-database/19/lnpls/CREATE-TRIGGER-statement.html#GUID-AF9E33F1-64D1-4382-A6A4-EC33C36F237B

Reply:
Sure, I was thinking the same thing because a trigger statement needs a 'body' function, but I'm unsure about where to put it.